### PR TITLE
docs: Configurations are placed in config dir

### PR DESCRIPTION
**Pull request by skovbakke**
Clarify in the README.md that the configuration files should be placed in a directory named `config`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,16 @@
 This is the repository for the configuration of a bare metal high available kubernetes cluster with load-balancing services supported. Outputs a tarball that can should be unpacked in the root of a linux filesystem. The cluster assumes a systemd cgroups are used as well as CRIO as the container runtime.
 
 ## Structure
-The entry point for the project is the make file in the root folder, which reads a config file for a full cluster setup.
+The entry point for the project is the `Makefile` in the root folder, which reads the `config` file, located in the `config` directory, for a full cluster setup.
```
Comment (jacobcrawford) [inline README.md:8]: The `config/config` structure has always been a source of confusion. Should we not change the folder name to "input", as everything in this folder is input for the script?

**Review by jacobcrawford — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MadsHT) [reply README.md:8]: I agree, We should either rename the config file og provide a path example:

```suggestion
The entry point for the project is the `Makefile` in the root folder, which reads the `config` file, located in the `config` directory, for a full cluster setup.

Files required in the config folder
`mukube-configurator/config/config`
`mukube-configurator/config/helm_requirements`
`mukube-configurator/config/values/*.yaml`

```

File: README.md [hunk 2]
```diff
@@ -5,14 +5,16 @@
...
 
-### [config](docs/config.md)
+After creating the configuration files the config partitions are generated by runnig `make`. Use `make clean` to get rid of artifacts.
 
-### helm_requirements.txt
+### [config/config](docs/config.md)
```
Comment (MadsHT) [inline README.md:12]: Maybe move this to the top, since the first part of this doc is about the config file?

**Review by MadsHT — COMMENTED** [review]
A few comments

File: README.md (hunk 1 above)
Comment (skovbakke) [reply README.md]: I have renamed the folder to input as requested

File: README.md (hunk 2 above)
Comment (skovbakke) [reply README.md]: I am not sure I understand what you are requesting. What should be moved?

**Comment by skovbakke** [comment]
I have renamed the config folder to input. Hopefully everything is more clear now 😀

**Review by jacobcrawford — APPROVED** [review]
LGTM 😸

**Comment by skovbakke** [comment]
Mads, do you want to look at this again?

**Review by MadsHT — APPROVED** [review]
LGTM, It's less confusing with `input` as folder name

**Pull request merged**